src/components/system/SystemAlert.jsx

```diff
@@ -76,6 +77,18 @@ class Alert extends Component {
       }];
     }
 
+    if (system.serviceWorkerUpdated) {
+      alertMessages.push({
+        message: `Thers is a newer version of this app`,
+        type: 'time-zone-offset-alert',
```

Comment:
I'm not sure what "type" is used for, but this is not a time zone alert, so if we need a type let's create a new one like "service-worker-alert"